content/source/docs/cloud/sentinel/manage-policies.html.md

```diff
@@ -42,19 +42,49 @@ In order to use Sentinel in Terraform Cloud, you'll first need to create a polic
 
 Every policy set requires a configuration file named `sentinel.hcl`. This configuration file defines:
 
-* Each policy that should be checked in the set
+* Each policy that should be checked in the set.
 * The [enforcement level](#enforcement-levels) of each policy in the set.
+* Any [modules](#modules) which need to be made available to policies in the set.
 
 The `sentinel.hcl` configuration file may contain any number of entries which look like this:
 
 ```hcl
 policy "terraform-maintenance-windows" {
-    enforcement_level = "hard-mandatory"
+  source            = "./terraform-maintenance-windows.sentinel"
+  enforcement_level = "hard-mandatory"
 }
 ```
 
 In the above, a policy named `terraform-maintenance-windows` is defined with a `hard-mandatory` [enforcement level](#enforcement-levels).
 
+#### Specifying a Policy Source
+
+A policy's `source` field that can point to a file within the policy repository, or a remote source, such as a policy from HashiCorp's [foundational policies library](https://github.com/hashicorp/terraform-foundational-policies-library).
+
+To specify a local source, make sure you prefix the `source` with a `./`, or `../`.
+
+```hcl
+policy "terraform-maintenance-windows" {
+  source            = "./terraform-maintenance-windows.sentinel"
+  enforcement_level = "hard-mandatory"
+}
+```
+
+To specify a remote source, supply the URL as the `source`. The following is taken as an example from the foundational policies library:
+
+```hcl
+policy "aws-cis-4.1-networking-deny-public-ssh-acl-rules" {
+  source            = "https://raw.githubusercontent.com/hashicorp/terraform-foundational-policies-library/master/cis/aws/networking/aws-cis-4.1-networking-deny-public-ssh-acl-rules/aws-cis-4.1-networking-deny-public-ssh-acl-rules.sentinel"
```

Comment:
Maybe worth making this point to a git reference instead of master branch? I worry about the file changing at some point in the future.